packages/@aws-cdk/core/test/integration.deps.readme

```diff
@@ -0,0 +1,5 @@
++--------------------------------------------------------------------------+
+| Since cdk-integ depends on cdk which depends on @aws-cdk/core (as a "dev |
+| dependency"), this integration test has been added to the package        |
+| @aws-cdk/aws-cloudformation under `test/integ.core-deps.ts`              |
++--------------------------------------------------------------------------+
```

Comment:
Nice, now when people point out to me in the future that they think they can't write integration tests because there's not a preexisting package in here, I can point at this.